open_src/influx/httpd/handler.go

```diff
@@ -351,9 +351,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer atomic.AddInt64(&statistics.HandlerStat.ActiveRequests, -1)
 	start := time.Now()
 
-	// Add version and build header to all InfluxDB requests.
-	w.Header().Add("X-Influxdb-Version", h.Version)
-	w.Header().Add("X-Influxdb-Build", h.BuildType)
+	// Add version and build header to all openGemini requests.
```

Comment:
The comments and codes must be consistent. Select either Geminidb or openGemini.